cfgov/housing_counselor/jinja2/housing_counselor/improved_index.html

```diff
@@ -75,17 +97,42 @@ <h1>Find a housing counselor</h1>
 
                     </div>
                     {% if zipcode %}
+                    {% if not zipcode_valid %}
+
+                    {% else %}
                     <div class="block">
                         <div class="results-header">
-                            <h2 class="h4">Displaying the 10 locations closest to ZIP code 52246</h2>
-                            <div class="action-buttons">
-                                <button class="a-btn a-btn__link">
-                                    Print list
-                                </button>
-                                <button class="a-btn a-btn__link">
-                                    Save list as PDF
-                                </button>
-                            </div>
+                            <ul class="m-list
+                                       m-list__horizontal
+                                       hud_hca_api_results_actions">
+                                <li class="m-list_item">
+                                    <button class="a-btn a-btn__link"
+                                       id="hud_print-page-link">
+                                        Print list
+                                        {{ svg_icon( 'print' ) }}
```

Comment:
This icon isn't in the current version of the tool and I'm not sure we have a super consistent way we apply link button icons like this across the site, so I'd vote for no icon for now.

Reply:
[It actually is](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/housing_counselor/templates/housing_counselor/index.html#L148) but isn't rendered. I'm guessing at some point it broke and no one noticed. However, let me know if in this modern incarnation it SHOULD have an icon?

Reply:
Interesting! This does seem like something that used to show up and then silently broke, so I say let's restore it. The only small change is that the icons should not be underlined (see the external link icon for an example).